src/cmd/reconcile.go

```diff
@@ -14,24 +13,27 @@ var reconcileResyncInterval int
 
 var reconcileCmd = &cobra.Command{
 	Use:   "reconcile",
-	Short: "Run in the foreground as a kubernetes controller to reconcile data with service entries in OpsLevel",
+	Short: "Run as a kubernetes controller to reconcile data in OpsLevel",
 	Long:  `Run in the foreground as a kubernetes controller to reconcile data with service entries in OpsLevel`,
 	Run: func(cmd *cobra.Command, args []string) {
+		var (
+			err    error
+			resync time.Duration = time.Hour * time.Duration(reconcileResyncInterval)
+		)
+		cobra.CheckErr(err)
 		config, err := LoadConfig()
 		cobra.CheckErr(err)
 
 		client := createOpslevelClient()
 		common.SyncCache(client)
-		resync := time.Hour * time.Duration(reconcileResyncInterval)
 		common.SyncCaches(createOpslevelClient(), resync)
 		queue := make(chan opslevel_jq_parser.ServiceRegistration, 1)
 		common.SetupControllers(config, queue, resync)
 		common.ReconcileServices(client, disableServiceCreation, queue)
-		opslevel_common.Run("Controller")
```

Comment:
@davidbloss and @rocktavious I would like to point out this change since it is easy to miss. I removed the call to [this signal handler from common.go.](https://github.com/OpsLevel/opslevel-common/blob/main/signal.go)

It is not being used and does not affect the application when I tophatted.

```
$ go run main.go service # has that line
2024-01-16T12:35:30-05:00 INF [/v1/pods] Informer is ready and synced
2024-01-16T12:35:30-05:00 INF [random-nginx-1] No changes detected to fields - skipping update
2024-01-16T12:35:30-05:00 INF [random-nginx-1] All tags already assigned to service.
2024-01-16T12:35:31-05:00 INF [random-nginx-2] No changes detected to fields - skipping update
2024-01-16T12:35:31-05:00 INF [random-nginx-2] All tags already assigned to service.
2024-01-16T12:35:31-05:00 INF [random-nginx-3] No changes detected to fields - skipping update
2024-01-16T12:35:31-05:00 INF [random-nginx-3] All tags already assigned to service.
^Csignal: interrupt
$ go run main.go service reconcile # commented out
2024-01-16T12:35:50-05:00 INF [/v1/pods] Informer is ready and synced
2024-01-16T12:35:51-05:00 INF [random-nginx-1] No changes detected to fields - skipping update
2024-01-16T12:35:51-05:00 INF [random-nginx-1] All tags already assigned to service.
2024-01-16T12:35:51-05:00 INF [random-nginx-2] No changes detected to fields - skipping update
2024-01-16T12:35:51-05:00 INF [random-nginx-2] All tags already assigned to service.
2024-01-16T12:35:51-05:00 INF [random-nginx-3] No changes detected to fields - skipping update
2024-01-16T12:35:51-05:00 INF [random-nginx-3] All tags already assigned to service.
^Csignal: interrupt
```

Reply:
I think doing this breaks signal handling when running inside a k8s pod where actual SIGINT and SIGTERM are sent to the process.  Doing a `ctrl + c` from the commandline is not the same thing as what k8s is doing to processes inside a container.

We should revert this.

Reply:
Will be addressed in https://github.com/OpsLevel/team-platform/issues/183